website/static/css/site.css

```diff
@@ -341,11 +345,17 @@ li.pointer {
 .comment-container {
     margin-top: 10px;
     margin-bottom: 10px;
+    border-left-style: dashed;
+    border-left-color: lightgrey;
```

Comment:
I don't think so since this CSS class is for the vertical dashed border to the left of each comment. Wouldn't using `text-muted` here make the comment text grey?